os-server/src/main/java/app/openschool/category/CategoryServiceImpl.java

```diff
@@ -156,4 +162,11 @@ public void delete(Long categoryId, Locale locale) {
     fileStorageService.deleteFile(oldFileName);
     categoryRepository.deleteById(categoryId);
   }
+
+  private void trowExceptionWhenTitleIsPresent(String title) {
+    boolean present = categoryRepository.findByTitle(title).isPresent();
+    if (present) {
+      throw new IllegalArgumentException("Title already exist");
```

Comment:
you need to create an error code and the actual test should be in messages.properties